examples/meta/generator/targets/cpp.json

```diff
@@ -147,7 +147,11 @@
             "create_parameter_observer": "create<ParameterObserver>($arguments)",
             "create_evaluation_result": "create<EvaluationResult>($arguments)",
             "create_distribution": "create<Distribution>($arguments)",
-            "create_combination_rule": "create<CombinationRule>($arguments)"
+            "create_combination_rule": "create<CombinationRule>($arguments)",
+            "create_string_features": "create<StringFeatures<uint16_t>>($arguments)",
+            "create_features_subset": "create<DenseSubsetFeatures<float64_t>>($arguments)",
+            "read_string_features": " create<StringFeatures<char>>($arguments)",
```

Comment:
why "read"?

Reply:
There are two `create_string_features`, one is `create<StringFeatures<uint16_t>>`, the other one is `create<StringFeatures<char>>`. as our meta translator don't support `function overloading`, so i rename one `create_string_features` to `read_string_features`.

Reply:
@karlnapf Could we pass the type as a function argument, like in numpy you would have `np.float32`, `np.float64` and so on?

Reply:
could do that. We do the same for dense features iirc

Reply:
it seems like that add the function argument still not fix the issue, our meta translator can't translate sg according to function argument, right?

Reply:
hmm I see. The issue is that StringFeatures is templated and it also inherits from Features (and there is template type overlap between the two). @karlnapf could we make StringFeatures untyped (or maybe give it a new untyped base class)?

Reply:
I dont think that would be good as densefeatures are kind of done similarly .... (for now)
Mmmmh, don't know ... what other options do we have?

Reply:
We could have `"create_string_features": " details::string_features($arguments)"`, but then we break the pattern

Reply:
what about `create_int_string_features` and `create_char_string_features` ?

Reply:
Yes, but shouldn't it be `char8`/`char16` or `uint8`\`uint16`?